common/src/main/java/tachyon/util/io/BufferUtils.java

```diff
@@ -249,9 +249,20 @@ public static boolean equalIncreasingByteBuffer(int start, int len, ByteBuffer b
    * @return ByteBuffer containing an increasing sequence of integers
    */
   public static ByteBuffer getIncreasingIntBuffer(int len) {
+    return getIncreasingIntBuffer(0, len);
+  }
+
+  /**
+   * Get a ByteBuffer containing an increasing sequence of integers starting at the given value.
+   *
+   * @param start the starting value to use
+   * @param len the target length of the sequence
+   * @return ByteBuffer containing an increasing sequence of integers
+   */
+  public static ByteBuffer getIncreasingIntBuffer(int start, int len) {
```

Comment:
Is this method added to be symmetrical with the other `getIncreasing...` methods?